plugin_interact.go

```diff
@@ -8,49 +8,43 @@ import (
 
 // A InteractionHandler holds information on how to handle a Minetest Interaction.
 type InteractionHandler struct {
-	Type        mt.Interaction // can be 255 to register on all interactions
-	Handler     func(*ClientConn, *mt.ToSrvInteract) bool
+	Type    Interaction
+	Handler func(*ClientConn, *mt.ToSrvInteract) bool
 }
 
+type Interaction uint8
+
+const (
+	Dig Interaction = iota
+	StopDigging
+	Dug
+	Place
+	Use
+	Activate
+	WildCard = 255
```

Comment:
It’s a bad idea to define a *global* wildcard this way. Make it more clear what it’s for, e.g. AnyInteraction or Interaction or Interact.